### ISSUE TITLE
Sorry for bothering,I think something wrong with your code!

### ISSUE THREAD

**Issue by twythebest** [NSSL-SJTU/HermesSim#13]
In the paper, W_{l,k}^{b,a} is the probability distribution for selecting negative samples, but in your code gnn_model.py, the following code:
loss = relu(sim_dists - self._loss_margin) + relu(1 - self._loss_margin - diff_dists[_i, _j])
in which diff_dists[_i, _j] selects this item as a negative sample based on _j, which is inconsistent with formulas (4) to (6) in your paper!
